sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/maskExpressions.scala

```diff
@@ -197,7 +197,7 @@ case class Mask(
    */
   override def eval(input: InternalRow): Any = {
     Mask.transformInput(
-      children(0).eval(input),
+      children.head.eval(input),
```

Comment:
In this case, the old one is more consistent to the next lines.

Reply:
You means the old one is better?

Reply:
Considering the characteristics and best practices of scala, personally, I feel the `.head` is better than `(0)`.